index.html

```diff
@@ -362,10 +365,11 @@
   - **`git checkout <name_of_branch>`**
   - **If there are uncommitted changes**:
       - If the branch is derived from the current branch and has no other changes they will follow you to the other branch
-      - If the other branch has diverted from the current, you will not be able to do change to the other branch..
-- Create new branch and switch to that
+      - If the other branch has diverged from the current branch, you will not be able to switch to it...
+- Create new branch and switch to it
   - **`git checkout -b <name_of_branch>`**
   - If there are uncommitted changes, those will be transferred to the new branch
+- Uncommitted (unstaged) changes actually affect the *working tree*, not the branch
```

Comment:
Since trees have not been discussed at all (only in the last part), I would prefer to avoid mentioning about trees here

Reply:
Understood.